builtin/providers/aws/resource_aws_api_gateway_method.go

```diff
@@ -134,6 +140,14 @@ func resourceAwsApiGatewayMethodUpdate(d *schema.ResourceData, meta interface{})
 		operations = append(operations, expandApiGatewayRequestResponseModelOperations(d, "request_models", "requestModels")...)
 	}
 
+	if d.HasChange("request_parameters_in_json") {
+		ops, err := expandApiGatewayMethodResponseParametersJSONOperations(d, "request_parameters_in_json", "requestParameters")
```

Comment:
and also there are two error messages inside the body of that function that should be changed accordingly too, so they're reflecting the reality.